### ISSUE TITLE
Additional ALPHAM multiplication in calculation of atmospheric water demand

### ISSUE THREAD

**Issue by julianrogger** [PIK-LPJmL/LPJmL#28]
In the calculation of the atmospheric water demand using the canopy conductance (following Monteith 1995) there is an additonal multiplication with ALPHAM (multiplying the conducatance scaling parameter GM with the maximum Priestley–Taylor coefficient ALPHAM).

For example L. 134 in water_stressed.c: 
  demand=(gp_stand>0) ? (1.0-*wet)*eeq*param.ALPHAM/(1+(param.GM*param.ALPHAM)/gp_stand) : 0;

In the Schaphoff et al. (2018) paper the demand function is reported as: 
D = (1-wet) * Eeq * ALPHAM / (1 + GM/gc), with gc being the potential canopy conductance under no water limitation. Where does this additional multiplication come from? 

Thanks in advance.

**Comment by cmueller-pik** [comment]
The Priestley-Taylor approach assumes a static factor (called ALPHAM here) for computing actual evapotranspiration from the equilibrium evapotranspriatoin (eeq) for substituting knowledge on atmospheric conditions such as VPD, see e.g. [here](https://en.wikipedia.org/wiki/Potential_evaporation#Priestley%E2%80%93Taylor)

**Comment by julianrogger** [comment]
Thanks for answering. I see the use of the ALPHAM factor to scale the equilibrium evapotranspiration but not necessarily why also the conductance scaling factor gm is being increased (param.GM * param.ALPHAM). In other parts of the code, versions without this additional scaling are commented out, so I assumed there is some reason why it was changed compared to the formulation reported in the paper. 

Thanks anyway.

**Comment by cmueller-pik** [comment]
Sorry for misunderstanding the question in the first place. I checked with my colleagues and this is what I learned: 
The equation used in the paper is a reduced (non-transparent) form of the original equation, which is both scaling `eeq` (numerator) and `GM` (denominator) with `ALPHAM`. In the [Schaphoff et al. 2018](http://dx.doi.org/10.5194/gmd-11-1343-2018) paper and the [Gerten et al. 2004](https://doi.org/10.1016/j.jhydrol.2003.09.029) paper (where this was introduced in equation 7), the parameter `gm` simply was already a combination of $`GM*ALPHAM`$.  
For the implementation of the nitrogen cycle in [von Bloh et al. 2018](http://dx.doi.org/10.5194/gmd-11-2789-2018), `ALPHAM` was recalibrated based on the original [Monteith 1995](https://doi.org/10.1016/0022-1694(94)05086-D) parameter range and to correctly account for the effect of `ALPHAM` on numerator and denominator, the nontransparent $`GM = GM * ALPHAM`$ was replaced. However, this wasn't documented leading to confusion now. Apologies for that.
